lib/ansible_tower_client/collection.rb

```diff
@@ -11,37 +11,47 @@ def all
     end
```

Comment:
@bdunne - I guess the more I think about this -  I would expect the all method to return a collection and in doing so would more closely match the `.all` method from active_record. 

I get now that we can act on the `Enumerator` class (`.each`, `.map` etc) - but in that case maybe we should rename `all` to something else?